From 713d471efd1459a1c6caa3c7e647eeda27862bcf Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Thu, 27 Feb 2014 02:20:34 -0500
Subject: installclass.py: skip the language selection

We only support the English atm, so skip the language selection.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 pyanaconda/installclass.py |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyanaconda/installclass.py b/pyanaconda/installclass.py
index d62ebd4..90a6710 100644
--- a/pyanaconda/installclass.py
+++ b/pyanaconda/installclass.py
@@ -149,6 +149,9 @@ class BaseInstallClass(object):
         # allow interface backends to skip certain steps.
         map(lambda s: dispatch.skip_steps(s), anaconda.intf.unsupported_steps())
 
+        # Skip the language selection
+        dispatch.skip_steps("language")
+
     # modifies the uri from installmethod.getMethodUri() to take into
     # account any installclass specific things including multiple base
     # repositories.  takes a string or list of strings, returns a dict
-- 
1.7.10.4

